graylog2-web-interface/src/views/components/sidebar/SideBar.jsx

```diff
@@ -2,17 +2,18 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 
-import { AddWidgetButton, SearchResultOverview } from 'views/components/sidebar';
+
+import { AddWidgetButton } from 'views/components/sidebar';
 import { Icon, Spinner } from 'components/common';
 import type { ViewMetaData } from 'views/stores/ViewMetadataStore';
 
-import styles from './SideBar.css';
+import { Container, ContentOverlay, SidebarHeader, Headline, ToggleIcon, HorizontalRuler } from './Sidebar.styles.jsx';
 import CustomPropTypes from '../CustomPropTypes';
 import HighlightingRules from './highlighting/HighlightingRules';
 import NavItem from './NavItem';
+import ViewDescription from './ViewDescription';
 
-const defaultNewViewTitle = 'New View';
-const defaultNewViewSummary = 'No summary.';
+const defaultNewViewTitle = 'New Search';
```

Comment:
If this is a file constant, what do you think of using an `ALL_CAPS` variable name? Or should those only be used on app wide constants?

Reply:
I would say the second one. Only for app wide constants. But I am open for arguments :smiley: